src/middlewared/middlewared/api/v25_10_0/acl.py

```diff
@@ -232,29 +317,44 @@ class DISABLED_ACLResult(DISABLED_ACL, AclBaseResult):
 
 class FilesystemGetaclResult(BaseModel):
     result: NFS4ACLResult | POSIXACLResult | DISABLED_ACLResult
+    """ACL information for the requested filesystem path."""
 
 
 class FilesystemSetAclOptions(BaseModel):
     stripacl: bool = False
+    """Whether to remove the ACL entirely and revert to basic POSIX permissions."""
     recursive: bool = False
+    """Whether to apply ACL changes recursively to all child files and directories."""
     traverse: bool = False
+    """Whether to traverse filesystem boundaries during recursive operations."""
     canonicalize: bool = True
+    """Whether to normalize and canonicalize ACL entries for consistency."""
```

Comment:
This is specifically about reordering the ACL entries so that they're in the order that Windows expects.